portal/src/graphql/portal/GetStartedScreen.tsx

```diff
@@ -247,6 +261,7 @@ function Card(props: CardProps) {
           to={internalHref}
           component={Link}
           className={styles.cardActionButton}
+          onClick={onClickActionButton}
```

Comment:
Per our discussion, link tracking should be done with Page View trigger or History Change trigger.